packages/apps/spaces/scripts/codegen-icons.js

```diff
@@ -16,6 +16,11 @@ const prettierConfig = JSON.parse(
   readFileSync(process.cwd() + '/.prettierrc', 'utf8'),
 );
 
+function getSvgViewBox(svgString) {
+  const match = svgString.match(/viewBox="([^"]*)"/);
+  return match ? match[1] : '0 0 24 24';  // return matched viewBox value or null if not found
+}
+
```

Comment:
This is not needed as all icons must have the same viewbox. SVG artwork like grids or other graphics should be treated manually and not like icons since we cannot really reuse artworks in many places.